projects/kit/components/segmented/segmented.directive.ts

```diff
@@ -15,9 +15,10 @@ import {RouterLinkActive} from '@angular/router';
 import {EMPTY_QUERY} from '@taiga-ui/cdk/constants';
 import {tuiQueryListChanges} from '@taiga-ui/cdk/observables';
 import {tuiInjectElement} from '@taiga-ui/cdk/utils/dom';
-import {filter, merge, switchMap} from 'rxjs';
+import {filter, merge, mergeAll, switchMap} from 'rxjs';
 
 import {TuiSegmented} from './segmented.component';
+import {tuiControlValue} from '@taiga-ui/cdk/observables';
```

Comment:
I think precommit should have moved it up to other imports

Reply:
thanks, corrected.